Lib/test/test_android.py

```diff
@@ -369,6 +371,8 @@ def write(b, lines=None, *, write_len=None):
                         ):
                             stream.write(obj)
 
+
+class TestAndroidRateLimit(unittest.TestCase):
```

Comment:
This test doesn't read its own output, so it doesn't need the `@skipIf` or the `setUp` from the other class.